_projects/2024/100535296/100535296.Rmd

```diff
@@ -0,0 +1,534 @@
+---
+title: "Age Distribution of Athletes by Sport since 1896"
+description: |
+  This project focuses on recreating a The Washington Post graph showing the olympic athlete distribution by sport and age since the inception of the modern games in 1896. 
+categories: "2024"
+author: "Irene Bosque Gala"
```

Comment:
Unfortunately, this template considers the last word as your surname, so it's better to define here "Irene Bosque-Gala".